File: dart-jaguar-client/lib/model/group_object.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

import 'package:swagger/model/meta_group_object.dart';
part 'group_object.jser.dart';

class GroupObject {
  
  final String name;
  
  final MetaGroupObject meta;
  
  final List<String> users;
  

  GroupObject(
    

{
    
     this.name = null,  
     this.meta = null,   this.users = const [] 
    
    }
  );

  @override
  String toString() {
    return 'GroupObject[name=$name, meta=$meta, users=$users, ]';
  }
}

@GenSerializer()
class GroupObjectSerializer extends Serializer<GroupObject> with _$GroupObjectSerializer {

}

File: csharp-dotnet2-client/src/main/CsharpDotNet2/IO/Swagger/Model/GroupObject.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class GroupObject : Dictionary<String, string> {
    /// <summary>
    /// Gets or Sets Name
    /// </summary>
    [DataMember(Name="name", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "name")]
    public string Name { get; set; }

    /// <summary>
    /// Gets or Sets Meta
    /// </summary>
    [DataMember(Name="_meta", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "_meta")]
    public MetaGroupObject Meta { get; set; }

    /// <summary>
    /// Gets or Sets Users
    /// </summary>
    [DataMember(Name="users", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "users")]
    public List<string> Users { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class GroupObject {\n");
      sb.Append("  Name: ").Append(Name).Append("\n");
      sb.Append("  Meta: ").Append(Meta).Append("\n");
      sb.Append("  Users: ").Append(Users).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public  new string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
